### PR TITLE
Add development and CI buildout configurations

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[isort]
```
Comment (thet) [inline setup.cfg]: isort in setup.cfg? i've never seen that, but i'm also not doing much CI setups.

Comment (hvelarde) [inline setup.cfg]: it works :-)

**Comment by thet** [comment]
instead adding the lengthy bootstrap.py you could just:
```
virtualenv .
pip install zc.buildout
./bin/buildout
```

travis should also be updated then.

i'd prefer that way instead of having the bootstrap.py boilerplate, which gets outdated with every new zc.buildout release.

**Comment by hvelarde** [comment]
that's not going to work as we have different versions of `zc.buildout` and `setuptools` specified in the `versions.cfg` file; the way I was doing it, works.

if you have make it work, I would love to see how.

File: setup.cfg (hunk 1 above)
Comment (jensens) [inline setup.cfg]: setup.cfg is best practice btw

**Pull request merged**